samples/msal-node-samples/ElectronSystemBrowserTestApp/README.md

```diff
@@ -4,7 +4,7 @@ An Electron application built with TypeScript that uses the MSAL Node library to
 
 ## Should I use this sample
 
-Understanding the way MSAL Node is used in this sample will help you if you're interested in building an Electron.js application to authenticate and acquire tokens using the system browser and custom URL scheme. Native apps that use the browser for authorization are more secure and can take advantage of the user's authentication session in the browser to enable single sign-on.
+Understanding the way MSAL Node is used in this sample will help you if you're interested in building an Electron.js application to authenticate and acquire tokens using the system browser. Native apps that use the browser for authorization are more secure and can take advantage of the user's authentication session in the browser to enable single sign-on.
```

Comment:
Can we rephrase as: This sample demonstrates how to integrate MSAL Node for authenticating your electron application....

Reply:
Done